src/server/pfs/server/driver.go

```diff
@@ -1051,25 +884,35 @@ func (d *driver) inspectCommit(ctx context.Context, commit *pfs.Commit, wait pfs
 			// Do nothing
 		}
 	}
-
+	if err := d.txnEnv.WithReadContext(ctx, func(txnCtx *txncontext.TransactionContext) error {
+		var err error
+		provCommits, err := pfsdb.CommitProvenance(context.TODO(), txnCtx.SqlTx, commit.Repo, commitInfo.Commit.ID)
```

Comment:
Why not use the context on line 887?

Reply:
that might be the right thing to do. Right now I exposed the context in the Provenance signatures but since in other places in the system we don't pass a context to DB calls, I've just filled them in with context.TODO() until we decide on an approach